test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+git+https://github.com/OCA/community-data-files@refs/pull/122/head#subdirectory=setup/account_tax_unece
```

Comment:
you MUST cleanup this before merging

Reply:
Is there any possibility for someone else to clean it? Because I've noticed that @koenloodts is inactive.

Reply:
@tbanevicius feel free to do it in a new PR :wink